connector_manager/manager/podigee_connector.py

```diff
@@ -0,0 +1,117 @@
+"""
+Handles Podigee OAuth operations, including token refresh and database updates.
+"""
+import mysql.connector
+import json
+from loguru import logger
+import os
+import requests
+
+def refresh_podigee_token(client_id, client_secret, refresh_token, redirect_uri=None):
+    """
+    Exchange a refresh token for a new access token and refresh token for Podigee.
+    
+    Args:
+        client_id (str): The client ID for the Podigee OAuth app
+        client_secret (str): The client secret for the Podigee OAuth app
+        refresh_token (str): The refresh token to exchange
+        redirect_uri (str, optional): The redirect URI. 
+                                    Defaults to https://connect.openpodcast.app/auth/v1/podigee/callback
+    
+    Returns:
+        dict: Token response containing access_token, refresh_token, etc.
+              Returns None if the request fails
+    """
+    if not redirect_uri:
+        redirect_uri = "https://connect.openpodcast.app/auth/v1/podigee/callback"
```

Comment:
You can have default values in the parameter list. Cleaner.

Reply:
jsut for now, we should move that to a config value somewhere anyways